test/mako/config/benchmark.go

```diff
@@ -25,39 +25,44 @@ import (
 
 	"github.com/golang/protobuf/proto"
 	mpb "github.com/google/mako/spec/proto/mako_go_proto"
+	"github.com/pkg/errors"
 )
 
 const koDataPathEnvName = "KO_DATA_PATH"
+const configMako = "/etc/config-mako"
 
 // MustGetBenchmark wraps getBenchmark in log.Fatalf
-func MustGetBenchmark() (*string, *string) {
-	benchmarkKey, benchmarkName, err := getBenchmark()
+func MustGetBenchmark() *mpb.BenchmarkInfo {
```

Comment:
We'll need to stage a PR in serving to pick up this signature change.

Reply:
serving is not calling this function directly, so it's not a breaking change.

Reply:
Yep just checked serving uses `Setup` method